packages/frontend/next-i18next.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  i18n: {
+    defaultLocale: 'uk',
+    locales: ['uk', 'en'],
+    localeDetection: false,
+  },
+  reloadOnPrerender: process.env.NODE_ENV === 'development',
```

Comment:
Is it used?

Reply:
do you mean reloadOnPrerender? 
"Because resources are loaded once when the server is started, any changes made to your translation JSON files in development will not be loaded until the server is restarted.
In production this does not tend to be an issue, but in development you may want to see updates to your translation JSON files without having to restart your development server each time."

Is it superfluous?

Reply:
Resolved. I thought it should be a part of i18n